generate/seccomp/seccomp_default_linux.go

```diff
@@ -0,0 +1,14 @@
+// +build linux
+
+package seccomp
+
+import "syscall"
+
+const (
+	CLONE_NEWIPC  = syscall.CLONE_NEWIPC
```

Comment:
`golint` [suggests `CloneNewIPC`][1], etc.  Breaking consistency with `syscall` would be unfortunate, but I'd prefer `CloneNewIPC` over hoop-jumping to make `golint` accept `CLONE_NEWIPC`.

[1]: https://travis-ci.org/opencontainers/runtime-tools/jobs/332713642#L537

Reply:
fair, but I was hoping to stick to `syscall` format